<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>product_testing</class>
 <widget class="QDialog" name="product_testing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="new_test">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>0</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>8</number>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string notr="true" extracomment="qqq">new</string>
   </property>
   <property name="icon">
    <iconset theme="new">
     <selectedon>:/icon/up-arrow.png</selectedon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="exportData">
   <property name="geometry">
    <rect>
     <x>1578</x>
     <y>530</y>
     <width>321</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Export data</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="test_proccess">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>1241</width>
     <height>901</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="title">
    <string>Tiến trình kiểm tra</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>1211</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>Kích hoạt</string>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tên bài test</string>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thông số test</string>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Kết quả</string>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="testSuite">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tên bài kiểm thử </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tên bài kiểm thử :</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="testProccess">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>780</y>
      <width>1201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>850</y>
      <width>1191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Thiết bị kiểm thử :</string>
    </property>
   </widget>
   <widget class="QLabel" name="testSuite_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MC</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Tài khoản</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Chọn bài kiểm thử</string>
   </property>
   <widget class="QComboBox" name="testsiuteList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tải xuống thiết bị</string>
    </property>
    <property name="icon">
     <iconset>
      <selectedon>:/icon/download-to-storage-drive.png</selectedon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>251</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Thông tin thiết bị</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1581</x>
     <y>61</y>
     <width>321</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="stop_testing">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Stop testing</string>
      </property>
      <widget class="QPushButton" name="SingleStop">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>101</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
                                border-style: solid;
                                border-width: 2px;
                                border-color: #1E90FF;
                                background-color: rgb(255, 0, 0);
                                padding: 5px;
                                }</string>
       </property>
       <property name="text">
        <string>Single Stop</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AllStop">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>101</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
                                border-style: solid;
                                border-width: 2px;
                                border-color: #1E90FF;
                                background-color: rgb(255, 0, 0);
                                padding: 5px;
                                }</string>
       </property>
       <property name="text">
        <string>All Stop</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Export data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="start_testing">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Start testing</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <widget class="QPushButton" name="SingleStart">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>31</y>
         <width>101</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
                                border-style: solid;
                                border-width: 2px;
                                border-color: #1E90FF;
                                background-color: rgb(0, 170, 127);
                                padding: 5px;
                                }</string>
       </property>
       <property name="text">
        <string>Single Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AllStart">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>101</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
                                border-style: solid;
                                border-width: 2px;
                                border-color: #1E90FF;
                                background-color: rgb(0, 170, 127);
                                padding: 5px;
                                }</string>
       </property>
       <property name="text">
        <string>All Start</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
